docs/components/loading-animation.md

```diff
@@ -19,8 +19,18 @@ eleventyNavigation:
 
 ## Usage
 
-{% include 'markup/spinner.njk' %}
+<div class="spinner-border text-info" role="status">
+  <span class="sr-only">Loading...</span>
+</div>
+<div class="spinner-border text-ui" role="status">
+  <span class="sr-only">Loading...</span>
+</div>
```

Comment:
Why get rid of the include statement? If this is too small of a file for this then there are many more that could follow this same presentation.

Reply:
it makes editing quicker as this is only used in once place. We can make those changes as we go forward.